package.json

```diff
@@ -1,92 +1,95 @@
-  "bin": {
-    "lunox": "./bin/lunox.cjs"
-  },
-  "types": "dist/index.d.ts",
-  "main": "dist/index.js",
-  "files": [
-    "dist/*",
-    "global.d.ts",
-    "stub/*"
-  ],
-  "scripts": {
-    "build": "npm run clean && NODE_ENV=production rollup -c && npm run fix",
-    "clean": "rm -rf dist & rm -rf bin",
-    "dev": "rollup -cw & nodemon -q -w dist dist/index.js",
-    "fix": "rm dist/Types.js",
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "lint": "prettier --check --plugin-search-dir=. . && eslint .",
-    "prettier": "prettier --write --plugin-search-dir=. .",
-    "format": "npm run prettier && npm run lint"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/kodepandai/lunox-framework.git"
-  },
-  "keywords": [
-    "laravel",
-    "nodejs",
-    "lunox",
-    "polka",
-    "svelte",
-    "mvc"
-  ],
-  "author": {
-    "name": "Akhmad Salafudin",
-    "email": "axmad386@gmail.com",
-    "url": "https://github.com/axmad386"
-  },
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/kodepandai/lunox-framework/issues"
-  },
-  "homepage": "https://github.com/kodepandai/lunox-framework#readme",
-  "dependencies": {
-    "@kodepandai/flydrive": "^2.0.0",
-    "@kodepandai/hash-equals": "^0.1.0",
-    "@kodepandai/node-input-validator": "^1.0.0-beta",
-    "colorette": "^2.0.16",
-    "commander": "^8.3.0",
-    "cookie": "0.4.1",
-    "dotenv": "^10.0.0",
-    "express-session": "^1.17.2",
-    "formidable": "^2.0.1",
-    "pluralize": "^8.0.0",
-    "polka": "^1.0.0-next.22",
-    "sirv": "^1.0.18"
-  },
-  "devDependencies": {
-    "@kodepandai/flydrive-s3": "^2.1.0",
-    "@rollup/plugin-json": "^4.1.0",
-    "@rollup/plugin-typescript": "^8.3.0",
-    "@types/bcryptjs": "^2.4.2",
-    "@types/cookie": "^0.4.1",
-    "@types/cors": "^2.8.12",
-    "@types/express-session": "^1.17.4",
-    "@types/formidable": "^2.0.0",
-    "@types/node": "^16.11.7",
-    "@types/pluralize": "^0.0.29",
-    "@types/polka": "^0.5.3",
-    "@typescript-eslint/eslint-plugin": "^5.3.1",
-    "@typescript-eslint/parser": "^5.3.1",
-    "bcryptjs": "^2.4.3",
-    "eslint": "^8.2.0",
-    "knex": "^0.95.14",
-    "mysql": "^2.18.1",
-    "nodemon": "^2.0.15",
-    "objection": "^3.0.0",
-    "prettier": "^2.4.1",
-    "rollup": "^2.61.1",
-    "rollup-plugin-multi-input": "^1.3.1",
-    "rollup-plugin-terser": "^7.0.2",
-    "session-file-store": "^1.5.0",
-    "svelte": "^3.44.3",
-    "tslib": "^2.3.1",
-    "typescript": "^4.5.4",
-    "vite": "^2.7.10"
-  },
-  "type": "module"
-}
+    "name": "lunox",
+    "version": "0.9.0",
+    "description": "Laravel-Flavoured NodeJs framework",
+    "bin": {
+        "lunox": "./bin/lunox.cjs"
+    },
+    "types": "dist/index.d.ts",
+    "main": "dist/index.js",
+    "files": [
+        "bin/*",
+        "dist/*",
+        "global.d.ts",
+        "stub/*"
+    ],
+    "scripts": {
+        "prepare": "npm run build",
+        "patch": "npm run patch",
```

Comment:
What is "patch" used for? Can we remove it?

Reply:
i think i was kinda high on caffeine when i write this one. xD